src/base.rs

```diff
@@ -49,7 +49,7 @@ impl PayloadId {
 /// Contains encoding symbols generated from a source block.
 ///
 /// As defined in section [4.4.2](https://tools.ietf.org/html/rfc6330#section-4.4.2).
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, PartialOrd, Eq, Ord, Serialize, Deserialize, Hash)]
```

Comment:
Same comment about deriving serialization

Reply:
See my reply for the comment above